traitsui/qt4/key_binding_editor.py

```diff
@@ -27,6 +27,7 @@
 
 
 from pyface.qt import QtCore, QtGui
+from pyface.api import YES, confirm
 
 from traits.api import Bool, Event
```

Comment:
nit - grouping the ETS imports together.

```suggestion
from pyface.qt import QtCore, QtGui

from pyface.api import YES, confirm
from traits.api import Bool, Event
```